components/analytics/overview/error-distribution-card.tsx

```
@@ -136,32 +308,144 @@ export function ErrorDistributionCard({
+              percentage: (currentCount / filteredErrorBots.length) * 100
+            })
+          }
+        } else {
+          // Not expanded, show as main type
+          chartData.push({
+            name: item.name,
+            value: currentCount,
+            percentage: (currentCount / filteredErrorBots.length) * 100
+          })
+        }
+      })
```

Biome (1.9.4)

[error] Prefer for...of instead of forEach.

forEach may lead to performance issues when working with large arrays. When combined with functions like filter or map, this causes multiple iterations over the same type.

(lint/complexity/noForEach)